src/redux/reducers/actionLiterals.ts

```diff
@@ -0,0 +1,9 @@
+export const actionLiterals = {
+    changePage: {
+        action: "page/changePage",
```

Comment:
This is not the action, but the so called action type. The action is the entire "change Page" object.